src/packet/packetlist.js

```diff
@@ -186,6 +190,8 @@ class PacketList extends Array {
 
   /**
    * Concatenates packetlist or array of packets
+   * @param {PacketList|Packet} packets - packets to concatenate
+   * @returns {PacketList} this packetlist
```

Comment:
I think we should also consider renaming this `concat` method, or return a new `PacketList` instance (like `Array.concat` does).